src/widgets/Tabs/VaccineSelect.js

```diff
@@ -94,6 +103,15 @@ const VaccineSelectComponent = ({
     [vaccines]
   );
   const runWithLoadingIndicator = useLoadingIndicator();
+
+  const wasVaccinatedToday = () => {
+    const currentDate = new Date();
+
+    return !!vaccinePatientHistory.filter(
+      history => history.confirmDate >= currentDate.getDate() - 1
+    ).length;
+  };
```

Comment:
I'm not sure of the requirement but when my daughter goes for her shots she gets like 3 vaccines in a day, so from my anecdotal evidence (the strongest form), getting multiple vaccines in a day is common?

Reply:
Hmm, yeah I think the requirement came in the context of COVID (where it's just one shot) but for vaccines in general that makes sense...maybe it would be better to check for duplicated vaccines then, rather than just that they have had a vaccine?

Reply:
Yeah you're right it can just be dismissed and whatever- either works for me and I'm sure if its a problem we can get feedback and change it 🤷